common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -12,21 +12,52 @@ import GlyphStyles from "../../utils/glyphs.js";
 
 type Props = {|
   +project: Project,
+  isSelectable?: boolean, // hovering over a card shows the select option it
+  onProjectSelect: ?Function,
 |};
 //fontawesome fixed width class
 const glyphFixedWidth = ' fa-fw'
 
-class ProjectCard extends React.PureComponent<Props> {
+type State = {|
+  isHovered: boolean,
+|}
+
+class ProjectCard extends React.Component<Props, State> {
   _cx: cx;
 
   constructor(): void {
     super();
+    this.state = {
+      isHovered: false,
+    };
     this._cx = new cx('ProjectCard-');
   }
 
   render(): React$Node {
     return (
-        <div className="ProjectCard-root">
+      <div
+        onMouseEnter={() => this.setState({ isHovered: true})} 
+        onMouseLeave={() => this.setState({ isHovered: false })}
+        className="col-12 col-lg-6"
+      >
+        <div className="ProjectCardRelative ProjectCard-root">
+          {this.props.isSelectable && this.state.isHovered && <div 
+          // { <div 
+
+            className="ProjectCardOverlay"
+          >
+            <div 
+              className="ProjectCardOverlayContainer"
+            >
+            </div>
+            <div
+              onClick={() => this.props.onProjectSelect && this.props.onProjectSelect(this.state.project)}
+              className="ProjectCardSelectButton"
+            >
+              Select
+            </div>
+          </div>
+          }
```

Comment:
This section is affecting all uses of ProjectCard, including the homepage. Adding this outer wrapper div with `classname="col-12 col-lg-6"` is causing render issues.

Reply:
addressed in 
aecf3bb
362466a